src/components/Navbar/Navbar.tsx

```diff
@@ -4,12 +4,12 @@ import "./Navbar.css";
 
 export function NavBar() {
     return (
-        <nav className="navbar-container">
+        <nav className="navbarContainer">
             <img src={logo} alt="Logo" />
             <div className="navbarItemsRight">
-                <NavLink to="/"><span className="navbarItem">Home</span></NavLink>
-                <NavLink to="/random"><span className="navbarItem">Random brewery</span></NavLink>
-                <NavLink to="/favorites"><span className="navbarItem">Favorites</span></NavLink>
+                <NavLink to="/" className="navbarLink"><span className="navbarItem">Home</span></NavLink>
+                <NavLink to="/random" className="navbarLink"><span className="navbarItem">Random brewery</span></NavLink>
+                <NavLink to="/favorites" className="navbarLink"><span className="navbarItem">Favorites</span></NavLink>
             </div>
```

Comment:
A good practice is to use the Prettier extension to format the code. In every project that I have been on, so far it is used. You can check how to set it up in VSCode (there is an extension for it) and format every file of your code. If you don't manage to do that, I will help